### ISSUE TITLE
Known Fingerprints are disabled after new certificate

### ISSUE THREAD

**Issue by Loader23** [ChatSecure/ChatSecure-iOS#983]
Hi,

when my certificate gets renewed the known fingerprints from my other devices dissappear until the client is online again. Then I have to manual enable these fingerprints again.
Any way to disable this? :-)

**Comment by chrisballinger** [comment]
are they marked as "removed by server"?

**Comment by Loader23** [comment]
No, they are just not visible until I start the other device (when it was offline) and then they are just disabled.

**Comment by mightyBroccoli** [comment]
I just had this with my fathers iphone. I have 3 keys, two of them were not visible and just came up after he sent a new message to me. But one was marked as removed by server. Was a real pain to get all keys activated again.

**Comment by chrisballinger** [comment]
We don't show keys in the device list if we haven't fetched the device
bundle (which has the public key). Perhaps we could show a placeholder item.

**Comment by Loader23** [comment]
When the placeholder item does not get deactivated, it could be the solution. :-)
